rts/Lua/LuaOpenGL.cpp

```diff
@@ -7050,14 +7052,20 @@ int LuaOpenGL::ObjectLabel(lua_State* L) {
 	return 0;
 }
 
-// https://registry.khronos.org/OpenGL-Refpages/gl4/html/glPushDebugGroup.xhtml
 /***
- * Pushes a debug marker for nVidia nSight 2024.04, does not seem to work when
- * FBO's are raw bound.
- * 
+ * Pushes a debug marker for debugging tools such as `nVidia nSight 2024.04`,
+ * see https://registry.khronos.org/OpenGL-Refpages/gl4/html/glPushDebugGroup.xhtml .
+ *
+ * May be unavailable and `nil` if the platform doesn't support the feature.
+ *
+ * Groups are basically named scopes similar to tracy's, and are pushed/popped independently
+ * from GL attribute/matrix push/pop (though of course makes sense to put them together).
+ *
+ * Tools are known to struggle to see the annotation for FBOs if they are raw bound.
```

Comment:
Is there some sort of `@bug` tag for documenting known caveats?

Reply:
Nope, but there is full markdown support so you can use whatever you like:
```c
* **WARNING**
* blah bah
```

Similarly you could have done for a hyperlink.
```c
see [glPushDebugGroup docs](https://registry.khronos.org/OpenGL-Refpages/gl4/html/glPushDebugGroup.xhtml).
```

You can use headings and everything if you think it's called for:

https://github.com/beyond-all-reason/spring/blob/d7637b4bc0400526485e3a78f121ae0c73f73707/rts/Lua/LuaVFS.cpp#L27-L134